cms/envs/common.py

```diff
@@ -1142,3 +1142,6 @@
 
 # OpenID Connect issuer ID. Normally the URL of the authentication endpoint.
 OAUTH_OIDC_ISSUER = 'https://www.example.com/oauth2'
+
+# 5 minute expiration time for JWT id tokens issued for Programs API.
```

Comment:
Tokens issued with this expiration time may also be used for requests made to the Credentials API. Something more general might go stale less quickly: "5 minute expiration time for JWT ID tokens issued by the CMS."